src/utils/numberFormatter.ts

```diff
@@ -0,0 +1,42 @@
+/*
+ * Copyright 2022 Harness Inc. All rights reserved.
+ * Use of this source code is governed by the PolyForm Shield 1.0.0 license
+ * that can be found in the licenses directory at the root of this repository, also available at
+ * https://polyformproject.org/wp-content/uploads/2020/06/PolyForm-Shield-1.0.0.txt.
+ */
+
+export interface NumberFormatterOptions {
+  truncate?: boolean
+}
+
+export const numberFormatter: (value?: number, options?: NumberFormatterOptions) => string = (
```

Comment:
Might be good idea to have some UTs to verify what this method is doing it actually does it.

Reply:
ok, will try to do